app/services/delete_account_service.rb

```diff
@@ -62,7 +63,7 @@ def call(account, **options)
   private
 
   def reject_follows!
```

Comment:
Wait, let me think... This made sense when this was SuspendAccountService, but do we actually *ever* need to do this when we're deleting now?

Reply:
Normally, this is already done in `SuspendAccountService`, but some codepaths skip this, such as blocking a domain altogether.

Reply:
OK, makes sense.